src/main/resources/kfsdbupgrade.properties

```diff
@@ -80,5 +80,4 @@ processed-files-file-name=kfs-database-upgrade-processed-files.txt
 # the upgrade at a specified point in the process in case of an error. 
 last-processed-file=
 
-# use 4 threads when converting maintenance documents
-mdoc.threads=4
+db.fragment=cryfdp9i1uh4.us-west-2.rds.amazonaws.com
```

Comment:
This should be verified that it will stay the same before PRD DB test runs occur. Will or Heather would be a good person to ask.

Reply:
This property is only used if the system property "db.name" is set (as a shorthand way to quickly switch between db instances)  In the "real" run the fully qualified JDBC url is embedded in the properties file.  I'll add a comment to that effect in the file.

Also, over in the "stability" oriented branch, the creation of a database instance is explicitly handled by the upgrade process itself, which makes it unnecessary to maintain configuration items like this, since the database endpoint is obtained and retained by the upgrade process itself and passed into the next phase of the ugprade.